ruddock/modules/rotation/routes.py

```diff
@@ -44,3 +50,34 @@ def update_comment(prefrosh_id):
   helpers.update_comments(prefrosh_id, flask.request.form.get("comments", ""))
   helpers.update_votes(prefrosh_id, flask.request.form)
   return flask.redirect(flask.url_for("rotation.show_prefrosh", prefrosh_id=prefrosh_id))
+
+@blueprint.route('/move/', methods=['POST'])
```

Comment:
If this is supposed to be the "home page" for the move functionality, make it a GET method, not POST.

GET is typically used for viewing state, POST is used for changing state.